SourceKitStressTester/Sources/StressTester/StressTester.swift

```diff
@@ -53,7 +53,13 @@ public class StressTester {
       if let timingsFile = options.requestDurationsOutputFile {
         // We are only keeping track of code completion durations for now.
         // This could easily be expanded to other request types.
-        let timing = AggregatedRequestDurations(instructionCounts: codeCompletionDurations.map({ $0.instructionCount }))
+        let timings: [Timing] = codeCompletionDurations.compactMap({ (request, instructionCount) in
```

Comment:
I'm fine with everything as it stands right now, but an alternative would be to change `codeCompletionDurations` to just `timings` (`[Timing]`) with `Timing` also including the request type. Including extra requests would just be a matter of adding a timing.

Reply:
I deliberately didn’t include the request type in timings to make the JSON file smaller by not repeating the request type over and over. We do support more request types by the inner key here:
https://github.com/apple/swift-stress-tester/blob/dd5b49095328b8d458b1bf30c59b5001f651f91c/SourceKitStressTester/Sources/StressTester/RequestDurationManager.swift#L75

Reply:
Fair enough 👍